packages/fast-components-styles-msft/src/call-to-action/index.ts

```diff
@@ -34,7 +31,10 @@ export const callToActionButtonOverrides: ComponentStyles<
     button__primary: {
         "&:hover": {
             "& $button_contentRegion": {
-                [applyLocalizedProperty("left", "right", Direction.ltr)]: "-4px",
+                left: (config: DesignSystem): string => {
```

Comment:
I noticed the animation of these is pretty janky also - switching to a transform makes it much smoother. If you don't want to change in this PR I can file a task for a follow up but thought it might be within scope - your call.

Reply:
May as well do with these changes. CTA is completely broken right now as it is, so I'd say it's definitely in scope